rsciio/_hierarchical.py

```diff
@@ -514,21 +527,22 @@ def _group2dict(self, group, dictionary=None, lazy=False):
                 dictionary[key] = value
         if not isinstance(group, self.Dataset):
             for key in group.keys():
-                if key.startswith("_sig_"):
+                if key.startswith("_ragged_shapes_"):
+                    # array used to parse ragged array, need to skip it
```

Comment:
This is related to https://github.com/zarr-developers/zarr-specs/issues/62  Support for n-d ragged arrays might be coming soon(ish)

Maybe add the reference there?

Reply:
I have put the link in the `_read_array` function.